website/content/api-docs/system/secrets-sync.mdx

```diff
@@ -371,7 +371,6 @@ Associations also contain the latest sync status for the secret they represent.
 $ curl \
     --header "X-Vault-Token: ..." \
     --request GET
-    --data @payload.json
```

Comment:
This is originally what brought me here - I assumed we don't need to send a payload with a `GET` request?